sdk/js/src/utils/consts.ts

```diff
@@ -513,6 +520,11 @@ const TESTNET = {
     token_bridge: undefined,
     nft_bridge: undefined,
   },
+  neutron: {
+    core: undefined,
```

Comment:
Same as above - the `wormhole-ibc` contract is deployed on Neutron testnet with address `neutron1rhqgnak5z0pla5sgqq7ykqh8glckk4h67fe0hgjfcyulx2nqa0yqaswtnp`